playbackSpeed/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+  id("voice.library")
+  id("voice.compose")
+  alias(libs.plugins.anvil)
+}
+
+anvil {
+  generateDaggerFactories.set(true)
+}
+
+android {
+  buildFeatures {
+    viewBinding = true
+  }
+}
+
+dependencies {
+  implementation(projects.common)
+  implementation(libs.material)
```

Comment:
If I understand correctly, this does not make use of the android components library?